docs/content/_index.md

```diff
@@ -1,608 +1,1007 @@
 ```
 
+By default OPA listens for HTTP connections on `0.0.0.0:8181`. See `opa run
+--help` for a list of options to change the listening address, enable TLS, and
+more.
+
+Inside of another terminal use `curl` (or a similar tool) to access OPA's HTTP
+API. When you query the `/v1/data` HTTP API you must wrap input data inside of a
+JSON object:
+
 ```json
 {
-  "result": {
-    "servers": [...],
-    "networks": [...],
-    "ports": [...],
-    "opa": {
-      "examples": {
-        "violations": [
-          {
-            "id": "s4",
-            "name": "dev",
-            "protocols": [
-              "http"
-            ],
-            "ports": [
-              "p1",
-              "p2"
-            ]
-          }
-        ],
-        "public_servers": [
-          {
-            "id": "s1",
-            "name": "app",
-            "protocols": [
-              "https",
-              "ssh"
-            ],
-            "ports": [
-              "p1",
-              "p2",
-              "p3"
-            ]
-          },
-          {
-            "id": "s4",
-            "name": "dev",
-            "protocols": [
-              "http"
-            ],
-            "ports": [
-              "p1",
-              "p2"
-            ]
-          }
-        ]
-      }
-    }
-  }
+    "input": <value>
 }
 ```
 
-In this case, we are only interested in the set of servers that violate the
-policy. We can use OPA's API to query for just those servers.
+Create a copy the input file for sending via `curl`:
 
-```http
-GET https://example.com/v1/data/opa/examples/violations HTTP/1.1
+```
+cat <<EOF > v1-data-input.json
+{
+    "input": $(cat input.json)
+}
+EOF
 ```
 
-...the response is the subset of the servers base document that use HTTP and are connected to a public network:
+Execute a few `curl` requests and inspect the output:
 
-```http
-HTTP/1.1 200 OK
-Content-Type: application/json
+```bash
+curl localhost:8181/v1/data/example/violation -d @v1-data-input.json -H 'Content-Type: application/json'
+curl localhost:8181/v1/data/example/allow -d @v1-data-input.json -H 'Content-Type: application/json'
 ```
 
-```json
-{
-  "result": [
-    {
-      "id": "s4",
-      "name": "dev",
-      "protocols": [
-        "http"
-      ],
-      "ports": [
-        "p1",
-        "p2"
-      ]
-    }
-  ]
+By default `data.system.main` is used to serve policy queries without a path.
+When you execute queries without providing a path, you do not have to wrap the
+input. If the `data.system.main` decision is undefined it is treated as an
+error:
+
+```bash
+curl localhost:8181 -i -d @input.json -H 'Content-Type: application/json'
+```
+
+You can restart OPA and configure to use any decision as the default decision:
+
+```
+./opa run --server --set=default_decision=example/allow
+```
+
+Re-run the last `curl` command from above:
+
+```bash
+curl localhost:8181 -i -d @input.json -H 'Content-Type: application/json'
+```
+
+### 5. Try OPA as a Go library
```

Comment:
Should we mention a 6th option of using https://play.openpolicyagent.org/ ?

Reply:
All of the policy examples should be openable in the playground. I think we should make the button more obvious and then this would be less necessary?

Reply:
Yea if we made the open in playground button more obvious that would definitely cover this.